plugins/module_utils/network/ios/argspec/snmp_server/snmp_server.py

```diff
@@ -453,7 +476,7 @@ class Snmp_serverArgs(object):  # pylint: disable=R0903
                             "type": "str",
                         },
                         "version_option": {
-                            "choices": ["auth", "access", "encrypted"],
+                            "choices": ["encrypted"],
```

Comment:
Wouldn't this break existing playbooks?

Reply:
And if just one value is valid for this option, maybe this should be `encrypted: True`? But again, that would be a breaking change.

Reply:
Resolving this as per discussion.